.github/workflows/sandbox.yaml

```diff
@@ -0,0 +1,51 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+name: Sandbox
+jobs:
+  check:
+    name: Sandbox
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - id: get-release
```

Comment:
It doesn't look like the output of this action is used anywhere?

Reply:
done